code/modules/mob/living/simple_mob/subtypes/animal/space/bats.dm

```diff
@@ -2,7 +2,7 @@
 	name = "space bat swarm"
 	desc = "A swarm of cute little blood sucking bats that looks pretty upset."
 	tt_desc = "N Bestia gregaria" //Nispean swarm bats, because of course Nisp has swarm bats
-	icon = 'icons/mob/bats.dmi'
```

Comment:
:-1: on unifying this. Different mobs may have different sets of sprites, so it doesn't necessarily make sense to group all [generic] mob sprites into a single file. We also don't have the infrastructure or expectations of clothing, so the differentiation in icon file comes from the assignment of the icon var itself. This is a lot less intuitive unless there's logical separation in the naming of the .dmi files used.

And I actually think our `icons/` folder hierarchy needs more depth. If you _must_ change this, make an animals dmi, and move this along with other animal simplemob sprites into it. Or create an animals subdir, and split other animal simplemob sprites off of the monolith. (Latter is very much preferred)

Yes, this is a different answer than for clothing, but clothing sprites are much more complicated, and adjusting them in a similar manner according to their existing infrastructure will be a fair bit more involved.